index.html

```diff
@@ -23,57 +27,58 @@
                 <path d="M4.646 4.646a.5.5 0 0 1 .708 0L8 7.293l2.646-2.647a.5.5 0 0 1 .708.708L8.707 8l2.647 2.646a.5.5 0 0 1-.708.708L8 8.707l-2.646 2.647a.5.5 0 0 1-.708-.708L7.293 8 4.646 5.354a.5.5 0 0 1 0-.708z"/>
               </svg>
             <ul class="header-link" id="header-link">
                 <li class="header-list-first"><a class="header-list-first-link" href="about.html">About</a></li>
                 <li class="header-list"><a  class="header-list-link" href="#">Program</a></li>
                 <li class="header-list"><a  class="header-list-link" href="#">Courses</a></li>
                 <li class="header-list"><a  class="header-list-link" href="#">News</a></li>
+                <li class="header-list"><a  class="header-list-link" href="#">Sponsors</a></li>
+                <li class="header-list"><a  class="header-list-link link-2" href="#">Free courses</a></li>
             </ul>
         </div>
     </header>
     <main class="main">
-        <div class="main-content">
-           <h2 class="main-content-title-2">Coding for bigenners !</h2>
-           <h1 class="main-content-title-1">Learn to code in <br>days without paying anything</h1>
-           <p  class="main-content-para">Lorem ipsum, dolor sit amet consectetur adipisicing elit. Velit accusantium impedit<br>
-             optio debitis eveniet?
-             Sunt id expedita nam asperiores dolores. <br>Hic laborum iusto praesentium excepturi beatae optio non quae deleniti
-            in? Sequi excepturi quisquam dolorem autem,<br> 
-            at quod reiciendis magnam culpa illo obcaecati, iusto ut dolor minus quibusdam ipsa qui.</p>
-           <h2 class="main-content-title-3"><a class="main-content-title-link" href="#">Apply now</a></h2>
-        </div >
-        <div class="program" id="lorems" >
-            <h2 class="program-title">Main Courses</h2>
+        <section class="main-content">
+            <div class="main-content-1">
+                <p class="main-content-title-2">Coding for beginners !</p>
+                <h1 class="main-content-title-1">LEARN TO CODE IN <br>DAYS WITHOUT PAYING ANYTHING</h1>
+                <p  class="main-content-para">knowledge is priceless, learn new technologies for a better future through this platform and at the same time learn to teach the technologies you have learned in order to increase your understanding .</p>
+                <p class="main-content-title-3"><a class="main-content-title-link" href="#">LEARN NEW FEATURES (2022)</a></p>
+                <p class="main-content-title-4">@Free courses.com</p>     
+            </div>
+         </section >
+        <section class="program" id="lorems" >
+            <h2 class="program-title">Main Courses<hr class="peaker-items-para-separate-1"></h2>
             <div class="program-item" id="program"></div>
             <a class="program-items-link" href="#">SEE THE WHOLE PROGRAM</a>
```

Comment:
- [ ] As requested by the previous reviewer your main program should look like the original one. I will make it clear to you. please implement it.
- [ ] There should be 5 cards but you have only four but you have a different background color. please stick to the design as much as possible.
### Your design
![your](https://user-images.githubusercontent.com/81727988/205036179-ca48f4f1-4bf4-4446-91fd-5df19cd15aa9.png)
### Figma design
![fimga](https://user-images.githubusercontent.com/81727988/205036228-defb9587-8a06-49eb-b8f7-a501ff7c5783.png)